frontend/app/openproject-app.js

```diff
@@ -203,7 +203,8 @@ openprojectApp
       $httpProvider.interceptors.push(function($q) {
         return {
           'request': function(config) {
-            if (!config.url.match('(^/templates|\\.html$)')) {
+            if (!config.url.match('(^/templates|\\.html$)') &&
```

Comment:
@ulferts I'd suggest simplifying the !&! to || and also adding a code comment for the future why it's an interceptor here (new people won't be aware OP can be run from a subfolder)